### PR TITLE
Edit/add some company

### PR DESCRIPTION
**Pull request by azinudinachzab**
Dear @ridwanbejo,
I added some company i've worked before. Kindly review, approve and merge my MR
Thanks and salam kenal :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -260,6 +261,7 @@ We believe that there are so many IT companies which are running their business
 - [Automate All](https://automateall.co.id/)
 - [HD Soluzion](https://www.hdsoluzion.com/)
 - [Sharing Vision](https://sharingvision.com)
+- [WGS](https://www.wgs.co.id)
```
Comment (ridwanbejo) [inline README.md]: Yang walden global service udah masuk mas  @azinudinachzab

File: README.md [hunk 2]
```diff
@@ -125,6 +125,7 @@ We believe that there are so many IT companies which are running their business
 - [Eudeka](https://www.eudeka.id/)
 - [Intelligo](https://intelligo.id)
 - [Alkademi](https://alkademi.id/)
+- [SanberCode](https://sanbercode.com)
```
Comment (ridwanbejo) [inline README.md:128]: Thank you mas sudah nambahin

File: README.md [hunk 3]
```diff
@@ -306,6 +308,7 @@ We believe that there are so many IT companies which are running their business
 - [OneMedic](http://onemedic.co.id/)
 - [Kaloli](https://kaloli.id/)
 - [Farmasee](https://farmasee.id/)
+- [Tweak](https://tweakindonesia.id/)
```
Comment (ridwanbejo) [inline README.md:310]: Thank you mas sudah nambahin

**Comment by ridwanbejo** [comment]
Salam kenal mas @azinudinachzab terima kasih sudah bantu nambahin direktorinya.

File: README.md (hunk 1 above)
Comment (azinudinachzab) [reply README.md]: ah so sorry, saya edit dlu

**Comment by azinudinachzab** [comment]
Done, removing WGS

File: README.md (hunk 1 above)
Comment (ridwanbejo) [reply README.md]: siap thanks mas.

**Comment by ridwanbejo** [comment]
> Done, removing WGS

Oke, saya merged yah mas @azinudinachzab 

Thanks for your contribution.

**Review by ridwanbejo — APPROVED** [review]
LGTM

**Comment by azinudinachzab** [comment]
Thank u

**Pull request merged**